playbook/app/pb_kits/playbook/pb_timeline/docs/_timeline_with_children.md

```diff
@@ -0,0 +1,4 @@
+You can use whatever kit you want for the date area of the timeline item, the node area, and the content area at the bottom or left.
+
+Checkout the code in this example to see the children kits in action.
+
```

Comment:
Let's update the copy to reflect the new names as well, once you change those.

Reply:
this has been upated